src/app/pages/submission-pages/submit-page/submit-page.component.html

```diff
@@ -4,11 +4,8 @@
   <section id="main-content-area" class="row padding-top-xsmall padding-bottom-xsmall" role="main">
     <h2 class="padding-top-xlarge">New data submission</h2>
 
-    <ul class="tabs" id="example-tabs">
-      <li *ngFor="let link of tabLinks" routerLinkActive="active is-active" class="tabs-title">
-        <a [routerLink]="link.href" routerLinkActive="active">{{link.title}}</a>
-      </li>
-    </ul>
+    <ebi-submission-menu *ngIf="activeSubmission?._links?.contents?._links" [submission]="activeSubmission"></ebi-submission-menu>
```

Comment:
Could `activeSubmission?._links?.contents?._links` be replaced by an Observable?

Reply:
This is fixed in the new commit and I get rid of this property